build.gradle

```diff
@@ -733,8 +741,8 @@ dependencies {
     compileOnly "org.opensearch:opensearch-job-scheduler-spi:${job_scheduler_version}"
     implementation "org.opensearch:common-utils:${common_utils_version}"
     implementation "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
-    implementation group: 'com.google.guava', name: 'guava', version:'31.0.1-jre'
-    implementation group: 'com.google.guava', name: 'failureaccess', version:'1.0.1'
+    compileOnly group: 'com.google.guava', name: 'guava', version:'31.0.1-jre'
```

Comment:
nit: these seemed to change in main in https://github.com/opensearch-project/anomaly-detection/pull/770. Do you know why this was never moved to 2.x?

Reply:
not sure. Maybe due to my mistakes forgetting to add the changes to build.gradle when backporting.